elliott

```diff
@@ -700,6 +700,46 @@ Example to add standard metadata to a 3.10 images release
             code=result.status_code))
         exit(1)
 
+#
+# Repair bugs
+# advisory:repair-bugs
+#
+@cli.command("repair-bugs", short_help="Find or add MODIFED bugs to ADVISORY")
+@click.option("--advisory", "-a",
+              required=True,
+              metavar='ADVISORY',
+              help="Repair bugs attached to to ADVISORY.")
+@click.option("--from", "original_state",
+              multiple=True,
+              default=['MODIFIED'],
+              type=click.Choice(elliottlib.constants.VALID_BUG_STATES),
+              help="Status of the bugs")
+@click.option("--to", "new_state",
+              default='VERIFIED',
+              type=click.Choice(elliottlib.constants.VALID_BUG_STATES),
+              help="Status of the bugs")
+@pass_runtime
+def repair_bugs(runtime, advisory, original_state, new_state):
```

Comment:
Needs more docstring. Exposition and examples plz.

Reply:
... quite. This was just to show the how easy the code was with the new library.